From 5e9adb7e58748f65994dbea37c531ce0e0dc129f Mon Sep 17 00:00:00 2001
From: Benutzer <benutzer@mai2014>
Date: Fri, 21 Aug 2015 21:29:40 +0200
Subject: Debian build dependencies.

---
 build/debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/debian/control b/build/debian/control
index d8925c0..43d625c 100644
--- a/build/debian/control
+++ b/build/debian/control
@@ -3,7 +3,7 @@ Section: sound
 Priority: optional
 Maintainer: Ubuntu MOTU Developers <ubuntu-motu@lists.ubuntu.com>
 XSBC-Original-Maintainer: RJ Ryan <rryan@mixxx.org>
-Build-Depends: debhelper (>= 4.0.0), pkg-config, docbook-to-man, xlibmesa-gl-dev, libglu1-mesa-dev, libqt4-opengl-dev | libqt4-dev (>= 4.6.0), scons, libjack-dev, portaudio19-dev, libid3tag0-dev, libmad0-dev, libogg-dev, libsndfile1-dev, libasound2-dev, libvorbis-dev, libsoundtouch1-dev, libmp4v2-dev, libfaad-dev, libportmidi-dev, libtag1-dev, libshout-dev, vamp-plugin-sdk, libprotobuf-dev, protobuf-compiler, libusb-1.0-0-dev, libchromaprint-dev, librubberband-dev, libopus-dev, libopusfile-dev, libsqlite3-dev
+Build-Depends: debhelper (>= 4.0.0), pkg-config, docbook-to-man, x11proto-gl-dev, mesa-common-dev, libglu1-mesa-dev, libqt4-opengl-dev | libqt4-dev (>= 4.6.0), scons, libjack-dev, portaudio19-dev, libid3tag0-dev, libmad0-dev, libogg-dev, libsndfile1-dev, libasound2-dev, libvorbis-dev, libsoundtouch1-dev, libmp4v2-dev, libfaad-dev, libportmidi-dev, libtag1-dev, libshout-dev, vamp-plugin-sdk, libprotobuf-dev, protobuf-compiler, libusb-1.0-0-dev, libchromaprint-dev, librubberband-dev, libopus-dev, libopusfile-dev, libsqlite3-dev
 Standards-Version: 3.7.3
 Homepage: http://www.mixxx.org/
 
-- 
2.1.4

